datafusion/tests/test_dataframe.py

```diff
@@ -730,9 +730,9 @@ def test_describe(df):
             "max",
             "median",
         ],
-        "a": [3.0, 3.0, 2.0, 1.0, 1.0, 3.0, 2.0],
-        "b": [3.0, 3.0, 5.0, 1.0, 4.0, 6.0, 5.0],
-        "c": [3.0, 3.0, 7.0, 1.7320508075688772, 5.0, 8.0, 8.0],
+        "a": [3.0, 0.0, 2.0, 1.0, 1.0, 3.0, 2.0],
+        "b": [3.0, 0.0, 5.0, 1.0, 4.0, 6.0, 5.0],
+        "c": [3.0, 0.0, 7.0, 1.7320508075688772, 5.0, 8.0, 8.0],
```

Comment:
Why are these changes needed?

Reply:
`null_count` was fixed upstream in https://github.com/apache/datafusion/pull/10260

The underlying data in the `df`.
```
>>> print(df)
DataFrame()
+---+---+---+
| a | b | c |
+---+---+---+
| 1 | 4 | 8 |
| 2 | 5 | 5 |
| 3 | 6 | 8 |
+---+---+---+
```